msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/ui/PrivateMessageManagerImpl.java

```diff
@@ -1499,7 +1599,21 @@ public void markMessageAsRepliedForUser(final PrivateMessage message) {
 				}
 			}
 		}		
-	} 
+	}
+
+	public void markMessageAsRepliedForUser(final PrivateMessage message, String userId) {				
+		  
+		PrivateMessage pvtMessage = getPrivateMessageWithRecipients(message);
+		List recipientList = pvtMessage.getRecipients();
```

Comment:
List should have a type

Reply:
This method is an overloaded version of an existing one in which the list also has no defined type.